app/script/calling/VideoGridRepository.js

```diff
@@ -35,26 +35,42 @@ z.calling.VideoGridRepository = class VideoGridRepository {
     const selfStreamState = callingRepository.selfStreamState;
     const calls = callingRepository.calls;
     this.grid = ko.observableArray([0, 0, 0, 0]);
-    this.selfId = ko.observable();
     this.thumbnailStream = ko.observable();
-    this.mirrorSelf = ko.pureComputed(() => !selfStreamState.screenSend());
 
     const selfStream = ko.pureComputed(() => {
-      return selfStreamState.videoSend() || selfStreamState.screenSend() ? localMediaStream() : undefined;
+      const stream = selfStreamState.videoSend() || selfStreamState.screenSend() ? localMediaStream() : undefined;
```

Comment:
I probably wrote this in the first place but perhaps parentheses around the condition would make things easier to read. 

I personally don't know which has the priority between the `||` and `?` operators, so I guess not everybody knows, this can lead to unexpected results

Reply:
I agree, but prettier decided that no parentheses are needed.

Reply:
I definitely agree. Just pull it out into its own variable.